pdp/__init__.py

```diff
@@ -56,7 +68,8 @@ def get_config():
             'js/pdp_vector_map.js'
             ], debug=(not config['js_min'])),
         'templates': resource_filename('pdp', 'templates'),
-        'version': get_distribution('pdp').version
+        'version': get_distribution('pdp').version,
+        'revision': "%s:%s" % (branch, sha)
```

Comment:
I'm not sure if this can be baked in to the package metadata at build time, but a [local version identifier](https://packaging.python.org/distributing/#local-version-identifiers) might be your best bet...